CMakeLists.txt

```diff
@@ -298,6 +305,14 @@ if (ETH2_SEPOLIA)
     add_definitions(-DETH2_SEPOLIA)
 endif()
 
+if (CACHE_SIZE_STATISTIC)
+    add_definitions(-DCACHE_SIZE_STATISTIC)
+endif()
+
+if (CACHE_SIZE_STATISTIC_MORE_DETAIL)
```

Comment:
add limit,should define CACHE_SIZE_STATISTIC firstly.

Reply:
it's not necessary to define CACHE_SIZE_STATISTIC first, define CACHE_SIZE_STATISTIC_MORE_DETAIL without CACHE_SIZE_STATISTIC also works.